webapp/locales/en.json

```diff
@@ -14,22 +14,22 @@
     "hello": "Hello"
   },
   "editor": {
-    "placeholder": "Leave your inspirational thoughts..."
+    "placeholder": "Write something inspiring..."
```

Comment:
Could you justify this change? I prefer the way it is

Reply:
Agree with @mattwr18 here